homeassistant/components/remember_the_milk/__init__.py

```diff
@@ -88,6 +99,10 @@ def _create_instance(hass, account_name, api_key, shared_secret,
         DOMAIN, '{}_create_task'.format(account_name), entity.create_task,
         description=descriptions.get(SERVICE_CREATE_TASK),
         schema=SERVICE_SCHEMA_CREATE_TASK)
+    hass.services.async_register(
```

Comment:
Use the sync version of register services.

Reply:
fixed